skrifa/src/outline/autohint/hint.rs

```diff
@@ -364,6 +364,14 @@ mod tests {
             .map(|point| (point.x, point.y))
             .collect::<Vec<_>>();
         assert_eq!(coords, expected_coords);
+        let expected_metrics = latin::HintedMetrics {
+            x_scale: 67109,
+            left_opos: 15,
+            left_pos: 0,
+            right_opos: 210,
+            right_pos: 192,
+        };
+        assert_eq!(metrics.unwrap(), expected_metrics);
     }
```

Comment:
We should have a test where the expected metrics is None?

Reply:
Added another test with space glyph which triggers this.